pkg/api/routes.go

```diff
@@ -102,8 +106,9 @@ func (rh *RouteHandler) SetupRoutes() {
 			rh.blobLockWrapper(rh.DeleteBlobUpload)).Methods("DELETE")
 		g.HandleFunc("/_catalog",
 			rh.ListRepositories).Methods("GET")
-		g.HandleFunc("/",
-			rh.CheckVersionSupport).Methods("GET")
+		g.HandleFunc("/", rh.CheckVersionSupport).Methods("GET")
+		//nolint (lll)
+		g.HandleFunc("/query", gqlHandler.GraphQL(search.NewExecutableSchema(search.Config{Resolvers: &search.Resolver{DB: rh.c.Config.Extension.Cve.cvedb, Cve: &cveinfo.CveInfo{Log: rh.c.Log}}})))
```

Comment:
break this into several lines? not possible? use nolint sparingly

Reply:
Ok, I will do that